src/parsing.jl

```diff
@@ -386,10 +386,10 @@ _DEFAULT_TYPE_MAP[:tstzrange] = Interval{ZonedDateTime}
 _DEFAULT_TYPE_MAP[:daterange] = Interval{Date}
 
 # Matches anything but the start or end of an interval or a comma
-const RANGE_ITEM = "[^\\[\\(\\]\\),]*"
+const RANGE_ITEM = raw"[^\[\(\]\),]*"
 # Makes sure the string starts and ends with a bracket or parentheses and a comma separates
 # the items in the interval
-const RANGE_REGEX = Regex("^([\\[\\(])($RANGE_ITEM),($RANGE_ITEM)([\\]\\)])\$")
+const RANGE_REGEX = Regex(raw"^([\[\(])(" * RANGE_ITEM * "),(" * RANGE_ITEM * raw")([\]\)])$")
```

Comment:
```suggestion
const RANGE_REGEX = Regex(string(raw"^([\[\(])(", RANGE_ITEM, "),(", RANGE_ITEM, raw")([\]\)])$"))
```
I think this is clearer, how about you?

Reply:
I'm also good with that